tests/lit-tests/ispc-library.ispc

```diff
@@ -0,0 +1,110 @@
+// This test verifies that API for ISPC C++ library works correctly
+// and without state corruption.
+
+// RUN: %{cxx} -x c++ -std=c++17 -I%{ispc_include} %s -L%{ispc_lib} -lispc -o %t.bin
```

Comment:
I think we need to allow having *.cpp files in lit-tests. It looks strange that we name files that are only C++ as .ispc.

Reply:
Oh, it's a good catch! Thanks!